### PR TITLE
Update touch handling, improve accessibility & add support for 3D Touch

### LINKED ISSUE #19: Provide support for 3D Touch
I believe the API just gives you a CGPoint so we may need to provide a public getter for the attributes at a point like the `-gestureRecognizerShouldBegin:` uses.

Need to make sure the gesture recognizers aren't firing at the same time, so we don't smash state around, since they explicitly depend on the UIGR flow for whether the NSLayoutManager & cousins live.

### LINKED ISSUE #21: Accessibility frames are off
Thanks of this great enhancement of UILabel!
While working with it I observed problems with the frames of the accessibility elements. So far I didn't found out the cause, but at least I found a typo in your code that might have something to do with it. But fixing the typo didn't help.
On ZSWTappableLabel.m method performWithLayoutManager in lines 185 and 193 there is a block defined/used called `sreenFrameForCharacterRange`. I think it should be named `screenFrameForCharacterRange`like the method's parameter does. 
As you can see in the screenshots the displayed accessibility frames do not fit the text elements.
Would be nice if you could have a look at it. Thanks!

**Edit:** This might be related to issue #20 .
simulator screen shot 12 07 2016 15 31 41
simulator screen shot 12 07 2016 15 31 49
simulator screen shot 12 07 2016 15 32 01
simulator screen shot 12 07 2016 15 32 10
simulator screen shot 12 07 2016 15 32 15

### LINKED ISSUE #23: Long press difficult to trigger on real device
The long press delegate method is difficult to trigger on a real device (vs. the simulator) because even the slightest movement of the finger while long-pressing a link will restart the 0.5 second timer.

Simply ignoring the UIGestureRecognizerStateChanged state in the longPress: method of ZSWTappableLabel appears to fix the issue.

### LINKED ISSUE #32: there is support to implement the tappableLabel in uitableviewcell
How can I implement a tappableLabel inside a uitableviewcell? 
Inside uitableviewcel always enable the didselectrow.

### PR DESCRIPTION
**Pull request by zacwest**
- Fixes #19 (3D Touch)
- Fixes #21 (Frames being odd especially when scrolling)
- Fixes #23 (LongPress had no wiggle room)
- Fixes #32 (Likely blocks UITableView's selection behavior now)

### CODE REVIEW

File: Example/ZSWTappableLabel/3D Touch/3DTouchObjectiveCViewController.m [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+@import Masonry;
+@import ZSWTappableLabel;
+@import SafariServices;
+
+@interface DDDTouchObjectiveCViewController() <ZSWTappableLabelTapDelegate, ZSWTappableLabelLongPressDelegate, UIViewControllerPreviewingDelegate>
+@property (nonatomic) ZSWTappableLabel *label;
+@end
+
+@implementation DDDTouchObjectiveCViewController
```
Comment (geoffmacd) [inline Example/ZSWTappableLabel/3D Touch/3DTouchObjectiveCViewController.m:19]: ha

File: Example/ZSWTappableLabel/3D Touch/3DTouchSwiftViewController.swift [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+//
+//  Created by Zac West on 4/20/19.
+//  Copyright © 2019 Zachary West. All rights reserved.
+//
+
+import UIKit
+import ZSWTappableLabel
+import SafariServices
+
+class DDDTouchSwiftViewController: UIViewController, ZSWTappableLabelTapDelegate, ZSWTappableLabelLongPressDelegate, UIViewControllerPreviewingDelegate {
```
Comment (geoffmacd) [inline Example/ZSWTappableLabel/3D Touch/3DTouchSwiftViewController.swift:13]: 3d works in swift, remember 📷ViewController ?

**Comment by geoffmacd** [comment]
👍

File: Example/ZSWTappableLabel/3D Touch/3DTouchSwiftViewController.swift (hunk 2 above)
Comment (zacwest) [reply Example/ZSWTappableLabel/3D Touch/3DTouchSwiftViewController.swift:13]: Hm, I think if it worked before it doesn't now. https://repl.it/repls/PoorEmotionalAutotote

> main.swift:1:7: error: class name can only start with a letter or underscore, not a number

**Pull request merged**